sdk/storage/azure-storage-blobs/CMakeLists.txt

```diff
@@ -2,24 +2,22 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required (VERSION 3.13)
+project(azure-storage-blobs LANGUAGES CXX)
 
-set (AZURE_STORAGE_BLOB_HEADER
-    inc/azure/storage/blobs.hpp
-    inc/azure/storage/blobs/append_blob_client.hpp
-    inc/azure/storage/blobs/blob_batch_client.hpp
-    inc/azure/storage/blobs/blob_client.hpp
-    inc/azure/storage/blobs/blob_container_client.hpp
-    inc/azure/storage/blobs/blob_options.hpp
-    inc/azure/storage/blobs/blob_responses.hpp
-    inc/azure/storage/blobs/blob_sas_builder.hpp
-    inc/azure/storage/blobs/blob_service_client.hpp
-    inc/azure/storage/blobs/block_blob_client.hpp
-    inc/azure/storage/blobs/page_blob_client.hpp
-    inc/azure/storage/blobs/protocol/blob_rest_client.hpp
-    inc/azure/storage/blobs/version.hpp
-)
```

Comment:
I don't understand, why don't you want to include header files?

Reply:
Is there any documents saying that it's a good practice?

Reply:
They don't need to be included, because source files include them. You don't build headers, you build sources.

Reply:
Is there any drawbacks to include them?

Reply:
No, other than that you probably need to update the list each time a file is added (but if you forget, nothing will fail).
Do you want to keep them? I can revert this specific change.

Reply:
The benefit of doing so is that we can see these header files in the project, so it's easier for us to navigate and edit. ![image](https://user-images.githubusercontent.com/52266153/101584029-ae6d7580-3a17-11eb-8796-42d7a90f9fae.png)

Reply:
I cannot see header files for `azure-core`.
![image](https://user-images.githubusercontent.com/52266153/101584110-d5c44280-3a17-11eb-90cd-fa6ba0695695.png)

Reply:
Thank you, I will revert, and add headers for core.

Reply:
Well, you don't have to do the same for core, as long as core developers are fine with current state. I don't want to act like meddling...

Reply:
@Jinming-Hu, you can change the project explorer view to see the entire source instead.
It's a personal choice at the end I think, I don't really like the cmake project view from VS. It goes away upon making changes to the CMakeLists files which is a shame hhahaha.

Reply:
Very good feedback, thank you @Jinming-Hu! Including all files when generating project file is a valuable scenario, I updated all the libraries to use this approach. I also checked for it to include all the files, and I ordered the list in the order you see them in IDE, so that it is easier to keep in sync/updated. Would you ok the changes now?

Reply:
yeah, I think this is perfect